framework/src/main/java/org/tron/common/application/TronApplicationContext.java

```diff
@@ -23,19 +22,13 @@ public TronApplicationContext(String... basePackages) {
   }
 
   @Override
-  public void destroy() {
```

Comment:
Shouldn't all the places where the destroy method is called be replaced, including all test code?

Reply:
destroy ->close -> doClose
<img width="699" alt="image" src="https://github.com/tronprotocol/java-tron/assets/82020050/3c238e67-cc6b-4337-a20f-8c138cf37037">